### DISCUSSION TITLE
Database changes in 0.12.11?

### DISCUSSION THREAD

**Discussion by tcroes** [gogs/gogs#7723] [Q&A]
We recently upgraded from 0.11.34 to 0.11.91 and then to 0.12.11 (in the same evening - in the event timing is an issue).  We also use an external LDAP authentication source.  We're using mysql.

We are seeing a 500 error when trying to set the LDAP authentication method as default:
Error 1054: Unknown column 'is_default' in 'field list'

We see the same error if we try to add a new LDAP auth method.  

Could we have missed a set of necessary database migration changes?

**Comment by stroucki** [comment]
same as 7224?
